src/harness/fakedatabase/exclusion_zone_db.json

```diff
@@ -0,0 +1,52 @@
+{
+  "terminated": false,
```

Comment:
This is not the format of the exclusion zone database.

Reply:
Modified   the exclusion record to have Geojson

Reply:
Please confirm your source for the exclusion zone DB format. I believe this is still incorrect.